### PR TITLE
Add web3bio extension

### PR DESCRIPTION
**Pull request by picturepan2**
## Description

[Web3.bio](https://web3.bio) is a platform for Web3 and Web 2.0 Identity Graph search and link in bio profiles. It provides a list of relevant identities when searching for a Twitter handle, Ethereum address, ENS domain, Lens profile, Farcaster account, Unstoppable Domains, and other Web3 identities.

Web3.bio for Raycast utilizes the Web3.bio Profile API to search for Ethereum (ENS), Lens, Farcaster, Unstoppable Domains, and other Web3 domains and identities.

## Screencast

Please see the `metadata` folder.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/web3bio/src/web3bio.tsx [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+        <PlatformFilter
+          platforms={(!profiles?.length || profiles.error ? [] : profiles)?.map((x: Profile) => x.platform)}
+          onSelectChange={setFilter}
+        />
+      }
+      actions={
+        <ActionPanel>
+          <Action title="Enter to Search" onAction={() => mutate()} />
+        </ActionPanel>
+      }
```
Comment (pernielsentikaer) [inline extensions/web3bio/src/web3bio.tsx:106]: Is this necessary?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you 🙂

- Can you give me a few example on profiles I can search for?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/web3bio/src/web3bio.tsx (hunk 1 above)
Comment (picturepan2) [reply extensions/web3bio/src/web3bio.tsx:106]: No. Removed.

**Comment by picturepan2** [comment]
Hi @pernielsentikaer Thanks for your review. [Web3.bio](https://web3.bio) is the Identity Explorer for Web3. You could try `vitalik.eth`, one of the founders of Ethereum, in the Raycast searchbox and the search results are all profiles that connected to his Web3 identity. Also, you can try searching for`nick.eth` or `brantly.eth` to see what profiles are connected to their Web3 identities. In the profile detail panel, you will see all their bio, contact info and social links there.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**